### PR TITLE
Visually indicate unsupported Templates & StandaloneVMs in Qube Manager

### PR DESCRIPTION
**Pull request by alimirjamali**
Fixes: https://github.com/QubesOS/qubes-issues/issues/4581

### CODE REVIEW

File: qubesmanager/qube_manager.py [hunk 1]
```diff
@@ -239,10 +243,16 @@ def update_power_state(self):
...
-                    manager_utils.get_feature(
+            if self.vm.klass in {'TemplateVM', 'StandaloneVM'}:
+                if manager_utils.get_feature(
                         self.vm, 'updates-available', False):
-                self.state['outdated'] = 'update'
+                    self.state['outdated'] = 'update'
+                elif manager_utils.get_feature(
+                        self.vm, 'os-eol', None):
+                    d = [int(x) for x in manager_utils.get_feature(self.vm, 'os-eol', '0-0-0').split('-')]
+                    eol = datetime(d[0], d[1], d[2])
```
Comment (marmarta) [inline qubesmanager/qube_manager.py:253]:
```suggestion
                        eol_string: str = vm.features.get('os-eol', '')
                        eol = datetime.strptime(eol_string, '%Y-%m-%d')
```

This is a bit more readable, but more importantly, it's the same logic as in the qui updater widget, to be sure we use the same one - that's the function https://github.com/QubesOS/qubes-desktop-linux-manager/blob/8bf07aa132dd47d1b501ec4ec2706c042f77dc7e/qui/utils.py#L88

**Review by marmarta — CHANGES_REQUESTED** [review]

File: qubesmanager/qube_manager.py (hunk 1 above)
Comment (alimirjamali) [reply qubesmanager/qube_manager.py]: Applied it and tested it. With one change that it should be `self.vm.features.get`

Comment (marmarta) [reply qubesmanager/qube_manager.py]: ah, right, you're correct

**Review by marmarta — APPROVED** [review]
pylint fail is due to some unrelated pylint update, I'll fix it independently

**Pull request merged**